### PR TITLE
Meta: Mark myself as a CODEOWNER of a bunch of paths

### PR DESCRIPTION
**Pull request by alimpfard**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
 /Userland/Libraries/LibWeb/WebDriver @linusg @trflynn89
 /Userland/Libraries/LibWeb/WebIDL @linusg
+/Userland/Libraries/LibXML @alimpfard
+/Userland/Services/RequestServer @trflynn89
 /Userland/Services/SQLServer @trflynn89
 /Userland/Services/WebDriver @trflynn89
+/Userland/Shell @alimpfard
 /Userland/Utilities/js.cpp @linusg
 /Userland/Utilities/sql.cpp @trflynn89
+/Userland/Utilities/wasm.cpp @trflynn89
```
Comment (drunderscore) [inline .github/CODEOWNERS:27]: 🤔 (and ditto the others too)

Comment (alimpfard) [reply .github/CODEOWNERS:27]: lol copy pasta

**Review by drunderscore — APPROVED** [review]
200% less copy pasta!

**Review by linusg — APPROVED** [review]
Impressive collection of things you've made there! 😮

**Pull request merged**